packages/react-native-client/ios/VideoPreviewView.swift

```diff
@@ -13,11 +17,8 @@ class VideoPreviewView: ExpoView {
         addSubview(videoView!)
     }
 
-    override func willMove(toSuperview newSuperview: UIView?) {
-        super.willMove(toSuperview: newSuperview)
-        if newSuperview == nil {
-            localVideoTrack?.stop()
-        } else {
+    private func trySetLocalCameraTrack() {
+        DispatchQueue.main.async {
```

Comment:
I think we should use `[weak self]` here. Or `[weak localVideoTrack, weak videoView]`

Reply:
https://www.swiftwithvincent.com/blog/when-do-you-really-need-to-use-weak-self
From this source i assume that using [weak self] might be better, although the chances that between calling `DispatchQueue.main.async` and its execution `VideoPreviewView` will be dealocated is marginal.